.github/renovate.json5

```diff
@@ -9,7 +9,7 @@
   "masterIssue": true,
   // bump for apps
   // update-lockfile for addons/libraries
-  "rangeStrategy": "bump",
+  "rangeStrategy": "update-lockfile",
```

Comment:
I was hoping this might help with keeping this more consistent across different open source CrowdStrike repos. Both [ember-toucan-styles](https://github.com/CrowdStrike/ember-toucan-styles/blob/main/.github/renovate.json5#L12) and [ember-toucan-core](https://github.com/CrowdStrike/ember-toucan-core/blob/main/.github/renovate.json5#L12) are using the "update-lockfile" strategy. I'm not too familiar with configuring renovate though, so let me know if this is a bad idea!

Docs:
https://docs.renovatebot.com/configuration-options/#rangestrategy

Reply:
Makes sense to me to keep these all consistent 👍